internal/command/versions.go

```diff
@@ -0,0 +1,48 @@
+package command
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/cyberark/summon/pkg/summon"
+	"github.com/cyberark/summon/provider"
+)
+//Create the AllVersions string to include versions of all the providers
+const SUMMON_VERSION = summon.VERSION
+var DefaultPath = provider.DefaultPath
+var AllVersions = GetVersions(GetProviderFileNames())
+
+//Run `$provider --version`(from DefaultPath),
+// 	for each provider in the list of providers
+//Then appends the responded versions to string and returns it
+func GetVersions(providers []string) string{
+	var toReturn = SUMMON_VERSION
```

Comment:
Avoid use of generic variable names especially for return types since someone looking at the bottom won't know what `toReturn` is. "What is this variable holding?" should guide you to a better name.